pixart.js

```diff
@@ -1 +1,11 @@
+var form = document.getElementById('form');
+var input = $("#color-field");
+var brush = $(".brush");
+var button = document.getElementById('set-color');
 
+var hello = function() {
+  event.preventDefault();
+  document.style.backgroundColor = 'red';
+  };
+
+button.addEventListener("click", hello);
```

Comment:
Continuing with the above comment, you would need to change `.addEventListener`'s second argument to an anonymous function that contains `event.preventDefault` along with a function call to `hello`.